test/functional/wallet.py

```diff
@@ -392,6 +392,8 @@ def run_test(self):
         extra_txid = self.nodes[0].sendtoaddress(sending_addr, Decimal('0.0001'))
         assert(extra_txid not in self.nodes[0].getrawmempool())
         assert(extra_txid in [tx["txid"] for tx in self.nodes[0].listtransactions()])
+        # The change output for extra_txid should be in the balance if we include unspendable
+        assert(self.nodes[0].getbalance() < self.nodes[0].getbalance(include_unspendable=True))
```

Comment:
Asserting on exact values can make the test more brittle. I think it's fine to test strict lesser than here.